File: ucl-ios-course-1/HelloWorld/HelloWorld/ViewController.m
//
//  ViewController.m
//  HelloWorld
//
//  Created by Nadiar Syaripul on 20/11/20.
//

#import "ViewController.h"

@interface ViewController ()

@property (weak, nonatomic) IBOutlet UILabel *helloLabel;

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view.
}

- (IBAction)helloButton:(id)sender {
    int height_feet = 5;
    int height_inches = 11;
    double height_cm = ((height_feet * 12.0) + height_inches) * 2.54;
//    self.helloLabel.text = [NSString stringWithFormat:@"%f", height_cm];
    
    if (height_cm > 200.0) {
        self.helloLabel.text = @"You are tall!";
    } else {
        self.helloLabel.text = @"Forget it";
    }
    
}

@end

File: Swift Fun/Swift Fun/ViewController.swift
//
//  ViewController.swift
//  Swift Fun
//
//  Created by Nadiar Ahmad Syaripul on 19/03/20.
//  Copyright © 2020 Nadiar Ahmad Syaripul. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    @IBOutlet weak var myLabel: UILabel!
    
    @IBOutlet weak var counter: UILabel!
    
    var initial = 1
    
    override func viewDidLoad() {
        
        super.viewDidLoad()
        counter.text = String(initial);
        // Do any additional setup after loading the view.
        
        // view.backgroundColor = UIColor.green
        
    }

    @IBAction func myButton(_ sender: Any) {
        myLabel.text = "Jabaja Funk Night"
    }
    
    @IBAction func increment(_ sender: Any) {
        initial = initial + 1
        counter.text = String(initial);
        
        if initial > 10 {
            view.backgroundColor = UIColor.gray
        }
        
    }
}

